src/lib/sqliteStore.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+const sqlite3 = require('sqlite3').verbose()
+
+function newSqliteStore (address) {
+  address = address || ':memory:'
+
+  let db = new sqlite3.Database(address)
```

Comment:
Should be `const`, not `let` because you're not going to reassign a different value later.